DistributedCollective/Sovryn-frontend
# Swap WS nodes with RPC nodes

task: https://taiga.sovryn.app/project/sovryn-light/us/321

QA: 
- need to check for every page and verify that data supposed to be read from blockchain is visible
- transactions can be send
